src/screens/TradingScreen/index.tsx

```diff
@@ -256,6 +258,14 @@ class Trading extends React.Component<Props, StateProps> {
                 break;
         }
     };
+
+    private handeDrag = (layout, oldItem, newItem) => {
+        layout.map(elem => {
```

Comment:
are we sure we want `.map` here?
map is used for other purposes

Reply:
yes, because, when we have top and bottom items, drag top one, then bottom moving to top of the screen and have a negative Y value, but this function return element, which was moved from top

Reply:
Use for or forEach